zip-0226.rst

```diff
@@ -177,21 +177,31 @@ The use of different value base points for different Assets enables the final ba
 Value Balance Verification
 --------------------------
 
-In order to verify the balance of the different Assets, the verifier MUST perform exactly the same process as for the Orchard protocol [#protocol-binding]_. 
+In order to verify the balance of the different Assets, the verifier MUST perform exactly the same process as for the Orchard protocol [#protocol-binding]_.
 
 For a total of :math:`n` Actions in a transfer, the prover MUST still sign the `SIGHASH` of the transaction using the binding signature key
-:math:`\mathsf{bsk} = \sum_{\mathsf{ \forall i\in \{1,...,n\}}} \mathsf{rcv_{i}}`.
+:math:`\mathsf{bsk} = \sum_{i=1}^{n} \mathsf{rcv_{i}}`.
 
 Then the verifier MUST compute
 
-.. math:: \mathsf{bvk = (\sum cv_i^{net})}  - \mathsf{ ValueCommit_0^{Orchard}(v^{balanceOrchard})} = \sum \mathsf{rcv_{i}^{net}}\mathcal{R}^{\mathsf{Orchard}}
+.. math:: \mathsf{bvk} = \mathsf{(\sum_{i=1}^{n} cv_i^{net})}  - \mathsf{ValueCommit_0^{OrchardZSA}(v^{balanceOrchard}, \mathcal{V}^{\mathsf{Orchard}})}
 
-and use it to verify the `bindingSignature` on the `SIGHASH` message.
+After computing :math:`\mathsf{bvk}`, the verifier MUST use it to verify the `bindingSignature` on the `SIGHASH` message.
 
 Rationale for Value Balance Verification
 ````````````````````````````````````````
 
-The main reason why no changes to the Orchard process are needed is that no Custom Assets can be unshielded, so all Custom Assets are contained within the shielded pool. This means that the net balance of the input and output values is zero, with only one Asset of value balance published, that of ZEC, :math:`\mathsf{v^{balanceOrchard}}`. No net amount of any other Asset will be revealed, and the number of Assets in the transaction is also hidden. The only exception to this is in the case that an Asset is *burnt*, as we will see below in the `burn mechanism`_.
+We assume :math:`n` Actions in a transfer. Out of these :math:`n` Actions, we further distinguish (for the sake of verbosity and clarity) between Actions related to ZEC and Actions related to Custom Assets. We assume :math:`l` Actions related to ZEC and :math:`n-l` actions related to Custom Assets, where :math:`l \in [0,n]`.
+
+The value balance verification is equivalent to:
+
+.. math:: \mathsf{bvk} = \mathsf{(\sum_{i=1}^{l} cv_i^{net}) + (\sum_{j=l+1}^{n} cv_i^{net})} - \mathsf{ValueCommit_0^{OrchardZSA}(v^{balanceOrchard}, \mathcal{V}^{\mathsf{Orchard}})}
```

Comment:
should be `(\sum_{i=1}^{l} cv_i^{net}) + (\sum_{j=l+1}^{n} cv_j^{net})}` (a small typo in `cv_j`)
Also, consider a different letter for the upper bound instead of `l`, maybe `k`

Reply:
> Also, consider a different letter for the upper bound instead of l, maybe k

Sure. I've used `m` (`k` is generally used in the indices along with `i`, `j` see, e.g. [Conventional variable names section](https://en.wikipedia.org/wiki/Variable_(mathematics)))